heat/core/linalg/tests/test_basics.py

```diff
@@ -61,19 +61,20 @@ def test_cross(self):
         self.assert_array_equal(cross_axisc, np_cross_axisc)
 
         # test vector axes with 2 elements
-        b_2d = ht.array(np_b[:-1, :, :], split=1)
-        cross_3d_2d = ht.cross(a, b_2d, axisa=1, axisb=0)
-        np_cross_3d_2d = np.cross(np_a, np_b[:-1, :, :], axisa=1, axisb=0)
-        self.assert_array_equal(cross_3d_2d, np_cross_3d_2d)
-
-        a_2d = ht.array(np_a[:, :-1, :], split=0)
-        cross_2d_3d = ht.cross(a_2d, b, axisa=1, axisb=0)
-        np_cross_2d_3d = np.cross(np_a[:, :-1, :], np_b, axisa=1, axisb=0)
-        self.assert_array_equal(cross_2d_3d, np_cross_2d_3d)
-
-        cross_z_comp = ht.cross(a_2d, b_2d, axisa=1, axisb=0)
-        np_cross_z_comp = np.cross(np_a[:, :-1, :], np_b[:-1, :, :], axisa=1, axisb=0)
-        self.assert_array_equal(cross_z_comp, np_cross_z_comp)
+        if np.lib.NumpyVersion(np.__version__) < "2.0.0b1":
+            b_2d = ht.array(np_b[:-1, :, :], split=1)
+            cross_3d_2d = ht.cross(a, b_2d, axisa=1, axisb=0)
+            np_cross_3d_2d = np.cross(np_a, np_b[:-1, :, :], axisa=1, axisb=0)
+            self.assert_array_equal(cross_3d_2d, np_cross_3d_2d)
+
+            a_2d = ht.array(np_a[:, :-1, :], split=0)
+            cross_2d_3d = ht.cross(a_2d, b, axisa=1, axisb=0)
+            np_cross_2d_3d = np.cross(np_a[:, :-1, :], np_b, axisa=1, axisb=0)
+            self.assert_array_equal(cross_2d_3d, np_cross_2d_3d)
+
+            cross_z_comp = ht.cross(a_2d, b_2d, axisa=1, axisb=0)
+            np_cross_z_comp = np.cross(np_a[:, :-1, :], np_b[:-1, :, :], axisa=1, axisb=0)
+            self.assert_array_equal(cross_z_comp, np_cross_z_comp)
```

Comment:
@Marc-Jindra During the PR meeting this morning we decided to keep the "cross_2d" functionality in Heat for now and compare it to the pytorch result in the tests when numpy==2. I.e. compare the distributed `ht.cross` (on split arrays) with the non-distributed pytorch output.